src/modules/ztsi/src/lib/Ztsi.cpp

```diff
@@ -205,97 +202,180 @@ bool Ztsi::IsValidConfig(const Ztsi::AgentConfig& config)
     return isValid;
 }
 
-int Ztsi::ReadAgentConfig(Ztsi::AgentConfig& config)
+bool Ztsi::FileExists(const std::string& filePath)
```

Comment:
Consider making this part of CommonUtils lib (this is identical from FileExists used in Settings, CommandRunner, etc). That change would replace all these separate instances with the common lib one.

We can do this as part of a separate PR later, your call.

Reply:
Thanks, I will do in a later PR and move `IsValidClientName()` at the same time.